helmfile/values/velero-sc.yaml.gotmpl

```diff
@@ -135,3 +135,13 @@ kubectl:
     runAsNonRoot: true
     runAsGroup: 10000
     runAsUser: 10000
+
+# Run velero-restore-helper init container with numeric UID and GID.
+configMaps:
+  restore-helper-config:
+    labels:
+      velero.io/plugin-config: ""
+      velero.io/pod-volume-restore: RestoreItemAction
+    data:
+      secCtxRunAsUser: "1000"
+      secCtxRunAsGroup: "1000"
```

Comment:
Thanks for identifying the problem, this looks good but did you consider using [secCtx](https://github.com/vmware-tanzu/velero/blob/v1.12.0/site/content/docs/v1.9/restic.md?plain=1#L410)? Seems more flexible

Reply:
I did but it wasn't clear to me that it wouldn't override the full security context or only the keys you set and I felt it was enough reading their code to understand the config for one day so I went with the safer option. :smile: